python/paddle/v2/dataset/flowers.py

```diff
@@ -61,6 +62,9 @@ def default_mapper(is_train, sample):
     '''
     img, label = sample
     img = load_image_bytes(img)
+
+    if is_train and random.randint(0, 1):
+        img = left_right_flip(img)
```

Comment:
The left_right_flip has already been applied in simple_transform:
https://github.com/PaddlePaddle/Paddle/blob/7333df8510953a0c99a1c0702666ff090c005378/python/paddle/v2/image.py#L325

Reply:
sorry, I ignored it